blob/blob.go

```diff
@@ -669,7 +669,15 @@ func (b *Bucket) newRangeReader(ctx context.Context, key string, offset, length
 
 // WriteAll is a shortcut for creating a Writer via NewWriter and writing p.
```

Comment:
Shouldn't the docstring mention that the MD5 checksum of `p` is computed each time and verified?
Also, could there be use cases where a caller might not want such a check to happen because, eg, a blob storage solution doesn't provide MD5 verification or uses another hash algorithm such as SHA256?

Reply:
> Shouldn't the docstring mention that the MD5 checksum of `p` is computed each time and verified?

Added a doc comment.

> Also, could there be use cases where a caller might not want such a check to happen because, eg, a blob storage solution doesn't provide MD5 verification or uses another hash algorithm such as SHA256?

No. Drivers that don't use MD5 verification already safely ignore this field as the portable Bucket type will already do the verification anyway. And if even if there were, it would be out of scope for this PR.